argo-cd-apps/base/member/infra-deployments/multi-platform-controller/multi-platform-controller.yaml

```diff
@@ -14,7 +14,9 @@ spec:
                 environment: staging
                 clusterDir: ""
           - list:
-              elements: []
+              elements:
+                - nameNormalized: stone-prod-m01
```

Comment:
the name of the cluster is stone-prd-m01, you need to minimally change the nameNormalized to stone-prd-m01 but to stay consistent, if you change clusterDir, you also need to rename the folder in component